s3_management/update_dependencies.py

```diff
@@ -22,6 +22,23 @@
         "nvidia-cudnn-cu12",
         "typing-extensions",
     ],
+    "torch-rocm": [
```

Comment:
I don't think these should be put under `torch-rocm`. Nothing in https://github.com/triton-lang/triton/blob/34a6a2ff86286bb41e4a776bc78d6e32ad48512f/python/setup.py#L580 is ROCm-specific. CUDA triton builds will need these dependencies too once `triton.txt` uses a new-enough commit of upstream triton that has this dependency.

Reply:
Sounds good changed description